bayesian/api_v1.py

```diff
@@ -263,7 +263,16 @@ def get(ecosystem, package, version):
             )
             raise HTTPError(400, msg)
         if ecosystem == 'maven':
-            package = MavenCoordinates.normalize_str(package)
+            try:
+                package = MavenCoordinates.normalize_str(package)
+            except Exception:
```

Comment:
Do we need to catch `Exception` here? I'd hope that MavenCoordinates.normalize_str will throw something like `ValueError`. Catching all exception could hide other errors in future and logs would be full of misleading information about invalid format.

Reply:
changed